Classes/KIFUIViewTestActor.m

```diff
@@ -263,9 +283,10 @@ - (void)enterTextIntoCurrentFirstResponder:(NSString *)text fallbackView:(UIView
 - (void)setText:(NSString *)text;
 {
     KIFUIObject *found = [self _predicateSearchWithRequiresMatch:YES mustBeTappable:NO];
```

Comment:
I had messed up the implementation of `setText:` though, so I just pushed a fix for it.